server.py

```diff
@@ -79,7 +80,15 @@ def make_video():
             logging.error(e)
             return make_response("unknown", 500)
 
-
+@app.route('/api/feedback', method = ['POST'])
+def get_feedbackvalue():
+    text = request.values.get('feedback')
+    if not text.strip():
+        return make_response("feedback empty", 400)
```

Comment:
`text`如果是`None`的話，呼叫`strip()`應該會報錯吧？
> 由於客戶端未傳送`feedback`參數，故`values.get(str)`預設回傳None，詳閱：[Class MultiDict.get(str)](https://werkzeug.palletsprojects.com/en/1.0.x/datastructures/#werkzeug.datastructures.MultiDict.get)。

應該要先判斷是否不為`None`，再對`text`做其他處理，才不會有伺服端錯誤(Http Status Code 500)。
```suggestion
    text = request.values.get('feedback')
    if text is not None:
        # Use text here
    else:
        return make_response("need feedback", 400)
```

Reply:
我的想法是讓他不要送空白的字串才這樣設計的如果  text = "" 用 if text is none就不會被擋下 來了 
```suggestion
    text = request.values.get('feedback')
    if not text.strip():
        return make_response("feedback empty", 400)
```

Reply:
我了解你的意思了剛剛有再去檢查過

Reply:
👍